_includes/default_roles/crier.md

```diff
@@ -1,19 +1,24 @@
-The _Crier_ reminds people in _Slack_ of the upcoming meeting.
+A _Crier_ reminds us in Slack of upcoming meetings.
 
-**1 Business Day Before Meeting**
+**One Business Day Before The Meeting**
 
-* [ ] Announce the meeting in our [Slack][chat]{:target="_blank"}.
-* [ ] Use the `Share` arrow icon to share that announcement with the [#general] channel.
+* [ ] *Announce* the meeting in our [Slack][chat]{:target="_blank"}.
+* [ ] *Send the announcement* to [#general] using the `Share` arrow.
 
 <img src="https://user-images.githubusercontent.com/9609562/220438340-2fed944a-142b-4217-bcae-5c0e0110ed05.png" width="500px" />
 
+**Behaviors**
+
+* Recruit external members.
+  * Present to the InnerSource community.
```

Comment:
What do you mean by this?

Reply:
A misfire.

I crossed wires with the Scribe role, which is a scribe behavior.

Deleted